autoware_launch/rviz/autoware.rviz

```diff
@@ -1213,81 +1281,86 @@ Visualization Manager:
-                        base_bounds_2: false
-                        current_vehicle_circles: false
-                        lateral_errors: false
-                        mpt_footprints: false
-                        vehicle_circle_lines: false
-                        vehicle_circles: false
-                      Topic:
-                        Depth: 5
-                        Durability Policy: Volatile
-                        History Policy: Keep Last
-                        Reliability Policy: Reliable
-                        Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/marker
-                      Value: true
-                    - Class: rviz_default_plugins/MarkerArray
-                      Enabled: true
-                      Name: ObstacleAvoidanceWall
-                      Namespaces:
-                        virtual_wall: true
-                        virtual_wall_text: true
-                      Topic:
-                        Depth: 5
-                        Durability Policy: Volatile
-                        History Policy: Keep Last
-                        Reliability Policy: Reliable
-                        Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/debug/wall_marker
-                      Value: true
+                      Name: VirtualWall
+                    - Class: rviz_common/Group
+                      Displays:
+                        - Class: rviz_default_plugins/MarkerArray
+                          Enabled: true
+                          Name: ObstacleStop
+                          Namespaces:
+                            {}
+                          Topic:
+                            Depth: 5
+                            Durability Policy: Volatile
+                            History Policy: Keep Last
+                            Reliability Policy: Reliable
+                            Value: /planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/debug/marker
+                          Value: true
+                        - Class: rviz_default_plugins/MarkerArray
+                          Enabled: true
+                          Name: SurroundObstacleCheck
+                          Namespaces:
+                            {}
+                          Topic:
+                            Depth: 5
+                            Durability Policy: Volatile
+                            History Policy: Keep Last
+                            Reliability Policy: Reliable
+                            Value: /planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/debug/marker
+                          Value: true
+                        - Class: rviz_default_plugins/MarkerArray
+                          Enabled: true
+                          Name: ObstacleAvoidance
+                          Namespaces:
+                            {}
```

Comment:
The `Namespaces` value is cleared. Is that okay?

Reply:
I modified from Rviz GUI, not directly editing the file. 
And the `namespaces` are empty for all messages too. It should be ok.

Reply:
IIRC, if the namespace is empty, markers are shown by default.
If somebody has checked the behavior, it's okay!

Reply:
Example: lanelet map markers with empty namespace:
![image](https://user-images.githubusercontent.com/21360593/170007594-e70e8ed1-f36e-4105-994d-e91b588536e8.png)

Reply:
Sorry, I mosunderstood.

Reply:
> if the namespace is empty, markers are shown by default.

There are two topics here: `debug_markers` and `virtual_wall_markers`.
All `debug_markers` are not displayed and all `virtual_wall_markers` are displayed by default.

Since it was troublesome to switch between displaying and not displaying at the namespace level (it would get weird when saving the rviz config), I did the separation at the topic level.

Reply:
Like this.
![image](https://user-images.githubusercontent.com/21360593/170010304-c922c314-a596-4886-83e9-ca61cfeaf452.png)